src/styles.css

```diff
@@ -163,18 +163,17 @@
 .card_languages,
 .languages_list {
   align-self: stretch;
-  display: flex;
+  /* display: flex;
   flex-direction: row;
-  justify-content: center;
```

Comment:
It is not always advisable to leave unused code in your projects. 
Linters may register as an error.